tests/test_integration.py

```diff
@@ -15,20 +15,36 @@ class TestNotify:
         }
     }
 
+    def set_email(self, email):
```

Comment:
Why did we need this new method? Do these methods need different email addresses?

Reply:
I think this was the solution discussed in tech session a while back. The different email addresses will return a different response from the API.

Reply:
Yeah since they are not running I would lean towards reverting this

Reply:
Reverted but kept comments on why the tests will not work till mock service is integrated